adversarial/README.md

```diff
@@ -0,0 +1,9 @@
+# Advbox
```

Comment:
`/adversarial/`  may not be a good path. However, we can just merge this PR, and change it later.

Reply:
OK. You can specify an appropriate directory. Thank you.